test/rendition.test.js

```diff
@@ -367,7 +367,7 @@ describe("rendition.js", () => {
             // overwrite path to point to test files
             rendition.path = './test/files/file.bmp';
             const result = await rendition.mimeType();
-            assert.strictEqual(result, 'image/x-ms-bmp');
+            assert.strictEqual(result, 'image/bmp');
```

Comment:
I'd cover this case this way, actually:

```suggestion
            assert.ok(result === 'image/bmp' || result === 'image/x-ms-bmp');
```

Wdyt?